src/ServiceProvider.php

```diff
@@ -29,33 +27,43 @@ public function boot(): void
 
     public function register(): void
     {
-        $client = fn () => new Client([
+        $this->registerClasses();
+    }
+
+    protected function createGuzzleClient(): Client
+    {
+        return new Client([
             'base_uri' => config('features.flagr_url')
```

Comment:
**suggestion:** probably set the timeout:
```suggestion
            'base_uri' => config('features.flagr_url'),
            'connect_timeout' => A_CONSTANT,
            'timeout' => A_CONSTANT
```

Reply:
Moved to config :)